packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -299,29 +289,31 @@ $table-unsorted-icon-color: get-color-token-from-bg(
 }
 
 @mixin usa-table--borderless {
-  thead {
-    th {
-      background-color: color($table-background-color);
-      border-top: 0;
-      color: color($table-text-color);
+  th,
+  thead td,
+  tfoot td {
+    background-color: color($table-background-color);
+  }
 
-      &[aria-sort] {
-        color: $table-sorted-header-text-color;
-      }
-    }
-    /* stylelint-disable selector-class-pattern */
-    th[data-sortable]:not([aria-sort]) {
-      .usa-table__header__button .usa-icon > g.unsorted {
-        fill: color($table-text-color);
-      }
+  th {
+    border-top: 1px solid color($theme-table-border-color);
+    color: color($table-text-color);
 
-      .usa-table__header__button:hover .usa-icon > g.unsorted {
-        fill: color(next-token($table-text-color, "darker"));
-      }
+    &[aria-sort] {
+      color: $table-sorted-header-text-color;
     }
-    /* stylelint-enable selector-class-pattern */
   }
+  /* stylelint-disable selector-class-pattern */
+  th[data-sortable]:not([aria-sort]) {
+    .usa-table__header__button .usa-icon > g.unsorted {
+      fill: color($table-text-color);
+    }
 
+    .usa-table__header__button:hover .usa-icon > g.unsorted {
+      fill: color(next-token($table-text-color, "darker"));
+    }
```

Comment:
Before/after, should the headers be darker on highlight since it's a `<th>`?

**Recently admitted US states (sortable table example)**
![image](https://github.com/user-attachments/assets/f97332ab-20a3-430a-b70b-125f252948e7)

Currently, it's the same color [accent cool lighter - `#e1f3f8`] as the other columns:
![image](https://github.com/user-attachments/assets/c773aeca-bc74-45a0-817a-c2330e3df792)


**Sortable striped table with various content types**
![image](https://github.com/user-attachments/assets/b8aba923-9c69-409e-b606-c57da1ad4931)